app/Insightly/Resources/InsightlyContactResource.php

```diff
@@ -55,4 +56,21 @@ public function delete(int $id): void
 
         $this->insightlyClient->sendRequest($request);
     }
+
+    /**
+     * @return int[]
+     */
+    public function findByEmail(string $email): array
```

Comment:
If we return only the `id` of a contact I think we need to make that more clear in the method name. Maybe something like `findContactIdsByEmail` or `findIdsByEmail`?